packages/codemods/nuxt/4/template-compilation-changes/__testfixtures__/fixture1.output.ts

```diff
@@ -0,0 +1,17 @@
+import { template } from 'lodash-es';
+import { readFileSync } from 'node:fs';
+addTemplate({
+    fileName: 'appinsights-vue.js',
+
+    options: {
+        /* some options */
+    },
+
+    getContents({ options: options }) {
+        const contents = readFileSync(resolver.resolve('zzzy.ejs'), 'utf-8');
```

Comment:
whats this zzzy thing? typo? the right one is './runtime/plugin.ejs' ?

Reply:
@alexbit-codemod I think we might want to create changes for any `.ejs` template files, I kept it as `zzzy.ejs` so that in case if any one accidently hardcode to make changes only on `./runtime/plugin.ejs` it could be caught in the tests, let me know if I should change it back to `./runtime/plugin.ejs`

Reply:
These are test fixtures. Input/before should get transformed to output/after once we run the codemod on it. The transformation logic should be captured in the codemod in the form of comments or you can have multiple test fixtures with different ejs files. Let's fix it.

Reply:
on it.